TUT/STA130F24_TUT02_Sep13.ipynb

```diff
@@ -2,101 +2,72 @@
-    "\n",
-    "1. Discuss the \"Monte Hall\" problem code with ChatGPT to make sure you understand how the code works and the problem works\n",
-    "2. See if ChatGPT can suggest any streamlining or readability or usability improvments to the code\n",
-    "3. Tell ChatGPT that you have code that creates a Markov chatbot and show it the code below and then use ChatGPT to make sure you understand how the code works and the problem works\n",
-    "    1. If ChatGPT asks how you will \"train\" your Chatbot you can tell it that you'll just use a series of stories with a lot of different characters\n",
-    "4. Tell ChatGPT that you have a couple extensions of the code to show it, and show it each of these of the algorithm in sequence\n",
-    "    1. Without giving it the answers, see if you can prompt ChatGPT to figure out the extensions in the code; namely, the trigram (rather than bigram) dependency, and the character specific Markov chains \n",
-    "    2. See if ChatGPT can suggest any streamlining or readability or usability improvments to the code\n",
-    "    3. See what ChatGPT is able to help you understand about the code and what's hard to understand even when  ChatGPTs is helping to explaining things\n",
-    "    4. Start a new ChatGPT and first show it the original Markov Chatbot, and then tell ChatGPT that you have an extension but just show it Extension #2 directly without showing it Extension #2 and see if it's still able to understand everything extension does. \n",
-    "5. Report on your experience with ChatGPT discussing how quickly it was able to be helpful for each of the above questions, and if so, how\n",
-    "6. Reflect on your experience interacting with ChatGPT and describe how your perception of AI-driven assistance tools in the context of learning coding, statistics, and data science may be evolving\n",
-    "7. By searching online and discussing the topic with ChatGPT, describe the relevance of learning and adaptability, communication, coding, and statistics and data analysis as skills in the modern world, epseically with respect to career opportunities (particularly in the context of the data science industry)\n",
-    "8. See if ChatGPT thinks you could be a statistician or a data scientist without coding or doing data analysis and summarise the assessment ChatGPT provides\n",
-    "9. Do some career exploration with ChatGPT to identify the skills that might be the most valuable for a career that you're interested in and paraphrase the conclusions of your conversation"
-   ]
+    "1. `for i in range(n)` , `for x in enumerate(lst)` and  `for i,x in enumerate(lst)`\n",
+    "2. `list()`, `dict()` and `tuple()`\n",
+    "3. NumPy functions:\n",
+    "    1. `import numpy as np`\n",
+    "    2. `np.array()`\n",
+    "    3. `np.random.choice([1,2,3])`\n",
+    "4. `if`/`else` conditional statements"
```

Comment:
Good -- yep this better is outlining -- I'm going to add one more level to outline and put "Review Monte Hall" here per my point 1 comment already noted; but, you'll hopefully be pleased to note that I use the word "Review" (just need to place it in different section)